src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Routes, Route, NavLink } from 'react-router-dom';
+
+import './TabsPage.scss';
+import { TabContent } from '../TabContent/TabContent';
+
+type Props = {
+  tabs: Tab[],
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => (
+  <>
+    <div className="tabs">
+      <h2>Business</h2>
+      <ul className="tabs__list">
+        {tabs.map(tab => (
+          <NavLink
+            className="tabs__item"
+            to={`/tabs/${tab.id}`}
+            key={tab.title}
```

Comment:
in most cases id is used as key

```suggestion
            key={tab.id}
```